feature/path-tracking/build.gradle.kts

```diff
@@ -0,0 +1,23 @@
+plugins {
+    alias(libs.plugins.mifos.android.feature)
+    alias(libs.plugins.mifos.android.library.compose)
+    alias(libs.plugins.mifos.android.library.jacoco)
+}
+
+android {
+    namespace = "com.mifos.feature.path_tracking"
```

Comment:
package name should not contains _, rename package name  "com.mifos.feature.path.tracking"

Reply:
Done